android/src/main/java/com/mobilepaymentssdkreactnative/Utils.kt

```diff
@@ -72,11 +73,11 @@ fun ReadableMap.readPaymentParameters(): PaymentParameters {
   locationId?.let { builder.locationId(it) }
   note?.let { builder.note(it) }
   orderId?.let { builder.orderId(it) }
-  processingMode?.let { builder.processingMode(it) }
   referenceId?.let { builder.referenceId(it) }
   statementDescription?.let { builder.statementDescription(it) }
   teamMemberId?.let { builder.teamMemberId(it) }
   tipMoney?.let { builder.tipMoney(it) }
+  idempotencyKey?.let { builder.paymentAttemptId(it) }
```

Comment:
It will cause a lot of confusion if we map `idempotencyKey` to `paymentAttemptId`. 
Instead, the ReadableMap should have a new field `paymentAttemptId` (only currently used by Android)
And the mapping in this class will become
```kotlin
  idempotencyKey?.let { builder.idempotencyKey(it) }
  paymentAttemptId?.let { builder.paymentAttemptId(it) }
```